x-pack/plugins/enterprise_search/public/applications/workplace_search/index.tsx

```diff
@@ -101,14 +105,22 @@ export const WorkplaceSearchConfigured: React.FC<InitialAppData> = (props) => {
         )}
       </Route>
       <Route path={PERSONAL_SOURCES_PATH}>
-        <PrivateSourcesLayout restrictWidth readOnlyMode={readOnlyMode}>
+        <PersonalDashboardLayout
+          restrictWidth
+          readOnlyMode={readOnlyMode}
+          sidebar={<PrivateSourcesSidebar />}
+        >
           <SourcesRouter />
-        </PrivateSourcesLayout>
+        </PersonalDashboardLayout>
       </Route>
       <Route path={PERSONAL_SETTINGS_PATH}>
-        <PrivateSourcesLayout restrictWidth readOnlyMode={readOnlyMode}>
+        <PersonalDashboardLayout
+          restrictWidth
```

Comment:
[Optional] See above comment on restrictWidth and just having it default to true vs passing it as a prop